### DISCUSSION TITLE
Winget can't read sources

### DISCUSSION THREAD

**Discussion by Kisdorf** [Devolutions/UniGetUI#2222] [Q&A]
Hello,

using wingetui with as a normal user (with or without adiministrator privileges) within Windows 11 Pro, wingetui doesn't show any packages installed with winget.
The paketmanager log says:

"[17.05.2024 08:59:59] Winget ▀▀▀▀▀▀▀▀▀▀▀
█  Executable: C:\Program Files\WingetUI\PackageEngine\Managers\winget-cli_x64\winget.exe
█  Arguments:  update --include-unknown  --accept-source-agreements
Failed when opening source(s); try the 'source reset' command if the problem persists.
An unexpected error occurred while executing the command: 
0x80070005 : Zugriff verweigert
[17.05.2024 08:59:59] Exit Code: -2147024891"

"source reset" or uninstalling and reinstalling wingetui doesn't solve it.
Using winget in a terminal (as normal user) or using wingetui within an administrator account is working without any problems.

Looks like it no problem with the sources but with some permissions within Windows 11.
What should i check / look at? 

Thank you.

Christoph

**Comment by marticliment** [comment]
This issue will be fixed on 3.1.0. Meanwhile, please use the "Reset WinGet Sources" under WingetUI Settings -> WinGet

**Reply by Kisdorf** [reply]
(Just for your information. I will wait for 3.1.0.:)
Thank you, but that also doesn't work for me. 
(Had already tried it and tried it now again.)

Thank you for your good work!
